src/components/OmsInstanceNavigator.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <!-- Todo: need to add translation to all the text once implemented in dxp-component -->
+  <ion-card>
+    <ion-card-header>
+      <ion-card-subtitle>
+        {{ "OMS instance" }}
+      </ion-card-subtitle>
+      <ion-card-title>
+        {{ authStore.getOms }}
+      </ion-card-title>
+    </ion-card-header>
+    <ion-card-content>
+      {{ 'This is the name of the OMS you are connected to right now. Make sure that you are connected to the right instance before proceeding.' }}
+    </ion-card-content>
+    <ion-button @click="goToOms(authStore.getToken.value, authStore.getOms)" fill="clear">
```

Comment:
Instead of directly accessing authStore in the state, make computed properties and access them here to avoid side effects and memory leaks in the template.